tokio-console/src/main.rs

```diff
@@ -127,6 +127,15 @@ async fn main() -> color_eyre::Result<()> {
                     .0
                     .push(Span::styled(" PAUSED", view.styles.fg(Color::Red)));
             }
+            let dropped_updates = state.async_ops_state().dropped_events()
+                + state.tasks_state().dropped_events()
+                + state.resources_state().dropped_events();
+            if dropped_updates > 0 {
+                header_text.0.push(Span::styled(
+                    format!(" dropped updates: {}", dropped_updates),
+                    view.styles.fg(Color::Red),
+                ));
+            }
```

Comment:
I think it's probably better to show a separate count for each category of dropped events. We may be in a state where we have not dropped any tasks, but we have dropped several async ops and several resources (for example). Summing all the counters hides these details.